include/SFML/Graphics/Text.hpp

```diff
@@ -372,8 +372,9 @@ class SFML_GRAPHICS_API Text : public Drawable, public Transformable
     ///
     /// This function computes the visual position of a character
     /// from its index in the string. The returned position is
-    /// in global coordinates (translation, rotation, scale and
-    /// origin are applied).
+    /// the global coordinates (translation, rotation, scale and
+    /// origin are applied) at the top of the character's text
+    /// line.
```

Comment:
I'd swap it around:

> ...position is at the top of the character's line in global coordinates...

Reply:
Is "character's text line" descriptive enough? If you hadn't had these discussions over the past few days, would you understand what it means?

Reply:
I'm thinking about adding some information to the SFML-Website repo with like an image that we can use to define:
"top of the line"
"bottom of the line"
"line height"
"glyph bounds"

Reply:
ascender height is very clear and googlable way of documenting what it's doing. I'm happy with this imho